packages/admin-ui/src/pages/system/actions.ts

```diff
@@ -44,7 +44,7 @@ export const changeEthClientTarget = (
       : false;
 
   await withToastNoThrow(
-    () => api.ethClientTargetSet({ target: nextTarget, deleteVolumes }),
+    () => api.ethClientTargetSet({ target: nextTarget, deletePrevEthClient }),
```

Comment:
If the users chooses to delete the package should we offer the option to keep volumes anyway?

Reply:
I would say it is a weird use case